traffic_router/core/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -297,9 +297,14 @@
 		<constructor-arg index="4" ref="TCPBlockingQueue" />
 	</bean>
 
+        <bean id="DnsStreamInetAddress" class="java.net.InetAddress" factory-method="getByName">
```

Comment:
looks like some white space differences for all the changes in this file. prob spaces vs tabs?

Reply:
changed spaces to tabs